libs/mixer/melody.ts

```diff
@@ -110,6 +110,60 @@ namespace music {
 
     let playToneID = 0
 
+    /**
+     * Play a melody from the melody editor.
```

Comment:
This comment should include a breakdown of the melody format; otherwise it might be tough to use this from javascript. Also, add descriptions for both of the parameters.